omnibus-mainnet.toml

```
@@ -64,3 +84,40 @@ defaultValue = "0xE561d5E02207fb5eB32cca20a699E0d8919a1476"
 
 [setting.ccip_token_pool]
 defaultValue = "0x057152DB365B47851B0A0bd431644b8eE21fE1b4"
+
+[setting.synthUsdeAsyncFixedFee]
+defaultValue = "0"
+
+[setting.synthUsdeSkewScale]
+defaultValue = "0"
+
+[setting.synth_usde_max_market_collateral]
+defaultValue = "<%= parseEther('100000000') %>"
+
++# we want to enable the spot market wrapper for usde
+[import.usde]
+source = "usde-token"
```

```suggestion
[import.usde]
source = "usde-token"
description = "we want to enable the spot market wrapper for usde"
```

Also do we need `target` by any chance?

---

target is only used on `provision` steps so no, we do not need `target`